packages/web3/src/connect-button/demos/name.tsx

```diff
@@ -0,0 +1,14 @@
+import { ConnectButton } from '@ant-design/web3';
+
+const App: React.FC = () => {
+  return (
+    <ConnectButton
+      address="0x21CDf0974d53a6e96eF05d7B324a9803735fFd3B"
+      name="wanderingearth.eth"
+      tooltip
+      connected
+    />
```

Comment:
这样会不会更好一点呀，更加贴近 Button 的用法
```suggestion
    <ConnectButton
      address="0x21CDf0974d53a6e96eF05d7B324a9803735fFd3B"
      tooltip
      connected
    >{wanderingearth.eth}</ConnectButton>
```

Reply:
感觉还是用 `name` 语义化更好一点。`children` 代表子元素，而在 `ConnectButton` 中子元素可能因为各种属性不同而不同，不太好直接用于 name。

Reply:
那也就是在我们的 ConnectButton 场景中，实际上 children 是没用的